uwb-beacon-firmware/src/main.c

```diff
@@ -46,7 +46,7 @@ int main(void)
     messagebus_init(&bus, &bus_lock, &bus_condvar);
     parameter_namespace_declare(&parameter_root, NULL, NULL);
 
-    blink_start();
+    //blink_start();
```

Comment:
Remove this?

Reply:
Initially turned off the blink thread because the LED was used to blink when the anchor advertises.